docs/our_work/a_and_e_forecasting_tool.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Accident and Emergency (A&E) Forecasting Tool'
+summary: 'A probabilistic model which gives a three-week forecast for A&E departments, predicting their expected admissions loads'
+origin: ''
+tags: ['MODELLING', 'HOSPITAL', 'MONTE CARLO', 'MCMC', 'URGENT CARE', 'ACCIDENT AND EMERGENCY']
+---
+
+The A&E forecasting tool models historical data from A&E departments to forecast likely admissions numbers over the coming days up to three weeks in advance. This is a probabilistic model, which provides confidence intervals within which a given location's admissions numbers are likely to fall. Breakdowns are provided at the Country, Region, ICS, Trust, and Site level as well as broad age groups. The tool is intended to be used alongside local information to help trusts and ICSs plan and prepare for the demand placed on their services. 
+
+Having this information can help departments, trusts, and ICS' plan and prepare for the demand placed on their services. Breakdowns are provided at the Country, Region, ICS, Trust, and Site level. Weekly trends are provided. 
+
+![text](../images/a_and_e_forecasting/weekly_trends.png)
```

Comment:
put alt text in the square brackets that describes the pictures for accessibility purposes!

Reply:
done!